js/genkit/tests/chat_test.ts

```diff
@@ -122,10 +124,11 @@ describe('chat', () => {
   });
 
   it('can start chat from a prompt', async () => {
-    const preamble = ai.definePrompt(
-      { name: 'hi', config: { version: 'abc' } },
-      'hi from template'
-    );
+    const preamble = ai.definePrompt({
+      name: 'hi',
+      config: { version: 'abc' },
+      messages: 'hi from template',
```

Comment:
😵‍💫  So wait, what does it mean to do `messages` as a string instead of `prompt`? I think I remember we talked about this, but don't remember the outcome.

Reply:
you can do something like `{{role "system"}} talk like a pirate {{ role "user" }} hi` in there.

Basically, before

```ts
definePrompt(
  {...},
  `{{role "system"}} talk like a pirate {{ role "user" }} hi`
)
```

after

```ts
definePrompt(
  {
     ...,
     messages: `{{role "system"}} talk like a pirate {{ role "user" }} hi`
  },
)
```

in the `prompt` option you can only put the "user"... can't put a system prompt in there.

```ts
definePrompt(
  {
     ...,
     prompt: `hi`
  },
)
```